sdl_android/src/main/java/com/smartdevicelink/managers/BaseSubManager.java

```diff
@@ -32,12 +32,12 @@ public BaseSubManager(@NonNull ISdl internalInterface){
 
 	/**
 	 * Starts up a BaseSubManager, and calls provided callback once BaseSubManager is done setting up or failed setup.
-	 * @param listener CompletionListener that is called once the BaseSubManager's state is READY or ERROR
+	 * @param listener CompletionListener that is called once the BaseSubManager's state is READY, LIMITED, or ERROR
 	 */
 	public void start(CompletionListener listener){
 		this.completionListener = listener;
-		if((state == READY || state == ERROR) && completionListener != null){
-			completionListener.onComplete(state == READY);
+		if((state == READY || state == LIMITED || state == ERROR) && completionListener != null){
```

Comment:
`state` should be accessed through the getState method. I would just add a line `int state = getState();` before the if statement